packages/gatsby-theme-store/src/sdk/orderForm/useRemoveItemAnalytics.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react'
```

Comment:
Why whould anyone ever want to remove an item without sending to analytics?

Reply:
I really don't know 😬  Maybe I am trusting the user too much.